xrpl/faucet.js

```diff
@@ -0,0 +1,74 @@
+const xrpl = require('xrpl');
+const { Command, Option } = require('commander');
+const { getWallet, getAccountInfo, sendTransaction } = require('./utils');
+const { addBaseOptions } = require('./cli-utils');
+const { loadConfig, printInfo, printWarn, getChainConfig } = require('../common');
+
+async function processCommand(chain, options) {
+    const wallet = await getWallet(chain, options);
+    const recipient = options.recipient || wallet.address;
+
+    if (wallet.address.toLowerCase() !== options?.recipient?.toLowerCase()) {
+        printInfo(`Requesting funds for`, recipient);
+    }
+
+    const client = new xrpl.Client(chain.rpc);
+    await client.connect();
+
+    try {
+        const balance = Number((await getAccountInfo(client, recipient)).Balance) / 1e6;
+        if (balance >= Number(options.minBalance)) {
+            printWarn('Wallet balance above minimum, skipping faucet request');
+            process.exit(0);
+        }
+    } catch (error) {
+        if (error.data.error !== 'actNotFound') {
+            printWarn('Failed to get account info for recipient', recipient);
+            throw error;
+        }
+    }
+
+    await client.fundWallet(wallet, { amount: "105" });
+    if (wallet.address.toLowerCase() !== recipient.toLowerCase()) {
+        const paymentTx = {
+            TransactionType: 'Payment',
+            Account: wallet.address,
+            Destination: recipient,
+            Amount: xrpl.xrpToDrops("100"), // TODO: Subtract actual fee.
```

Comment:
```suggestion
            Amount: xrpl.xrpToDrops((amount - FEE).toString()), // TODO: Subtract actual fee.
```
create a `FEE` constant instead of hardcoding